packages/field-plugin/src/messaging/pluginMessage/pluginToContainerMessage/PromptAIMessage.ts

```diff
@@ -0,0 +1,87 @@
+import { hasKey } from '../../../utils'
+import {
+  isMessageToContainer,
+  type MessageToContainer,
+} from './MessageToContainer'
+
+export type PromptAIAction =
+  | 'prompt'
+  | 'complete'
+  | 'shorten'
+  | 'extend'
+  | 'rephrase'
+  | 'summarize'
+  | 'simplify'
+  | 'translate'
+  | 'tldr'
+  | 'adjust-tone'
+  | 'emojify'
+  | 'fix_spelling_and_grammar'
+
+export const promptAIActionsList: PromptAIAction[] = [
```

Comment:
nit: no way of extracting the type from the const as we did for the meta tags in the new plugin?